.golangci.yml

```diff
@@ -10,7 +10,9 @@ linters:
     - gofmt
     - goimports
     - gosec
-    - megacheck
+    - gosimple
+    - staticcheck
+    - unused
     - misspell
     - nakedret
     - nolintlint
```

Comment:
I would like to suggest to configure nolintlint stricter 

https://golangci-lint.run/usage/linters/#nolintlint

Having `nolint: revive` added without explanation is a bit a problem because then you have no idea why it was added. revive linters has so many rules...

Of course, this could be part of another PR.

I have no idea about the impact of such changes on the existing nolint directives present in the code.

Reply:
@ccoVeille thx for the feedback, I'll check how to make it stricter and I'll add a comment to explain the reason (BTW: we can avoid it if we replace the var name with `_`, but I prefer to keep the same name used in cobra)

Reply:
yeah I don't understand the point of `//nolint:revive`?

Reply:
Ok ran the build - I see. I think I'd prefer to rename the variables and not have an exception 👍🏼